.travis.yml

```diff
@@ -43,6 +43,7 @@ install:
   - go mod vendor
   - git diff --exit-code vendor/
   - go install -v -race -mod=vendor ./...
+  - GOOS=darwin GOARCH=arm64 go install -v -mod=vendor ./...
```

Comment:
Any reason not to pass `-race` here, like we do in the amd64 build on the line above?

Reply:
Yes, it would seem that cgo is required for `-race`, which is not available with cross-compilation.

In view of this pipeline being eventually migrated to GitHub Actions, it seemed like cross-compiling was preferable to running a separate macOS-native build.